README.md

```diff
@@ -1,223 +1,299 @@
 $ export AWS_DEFAULT_REGION=us-west-2
 ```
-Note: The role of the AWS user should have permissions to create all of the resources provided in `/modules`.
-4. Set the environment variables. See `example.env`.
+
+Note: The role of the AWS user should have permissions to create all
+of the resources provided in `/modules`.
+
+It's worthwhile at this point to confirm that AWS CLI is authenticated
+properly. Try running this command `aws sts get-caller-identity`. If
+this works, you're probably set to move forward.
+
+
+3. There are a few environment variables that should be set before
+   provisioning the infrastructure. Edit `example.env` based on your
+   requirements and then run the commands below to set your local
+   environment variables.
+
 ```
-set -a       
+set -a
 source example.env
 set +a
 ```
-5. Run `terraform init`. Terraform downloads and installs the provider plugins, which are used to interact with AWS and initializes the backend to store the state file.
+
+You can quickly double check that your environment variables have been
+set by running `env | grep -i tf_var` to double check that the
+environment variables match what you specified in `example.env`
+
+The full list of variables that can be configured are in
+[variables.tf](./variables.tf).
+
+4. Now we'll run `terraform init` to download and install the provider
+   plugins, which are used to interact with AWS and initializes the
+   backend to store the state file.
+
 ```
 terraform init
 ```
-5. Run `terraform plan`. Terraform compares the desired state (as defined in your Terraform configuration files) to the current state of your infrastructure and determines what actions need to be taken to achieve the desired state.
+
+5. Now we're going to plan and apply using Terraform. While working
+   iteratively on your supernet, you'll typically make changes to the
+   Terraform modules, then plan to see what changes will be made to
+   your infrastructure, and then finally apply if the plan looks good.
+
 ```
 terraform plan
+terraform apply
 ```
-6. Run `terraform apply`. Terraform creates, modifies, or deletes resources as necessary to achieve the desired state.
 
-```
-terraform apply -auto-approve
-```
-7. Save `terraform output pk_ansible` to a file. And change permissions so that only the owner of the file can read and write to the file.
+6. During the Terraform apply, a private key was generated to give you
+   access to your VMs. You'll need to shave that key using the
+   following commands.
+
 ```
 terraform output pk_ansible > ~/devnet_private.key
-chmod 600 ~/devnet_private.key 
-eval "$(ssh-agent)"
-ssh-add ~/devnet_private.key 
+chmod 600 ~/devnet_private.key
 ```
 
+At this point all of the AWS infrastructure needed to run a supernet
+should be deployed. This is a good time to login to your AWS Console
+and take a look.
+
 ## Ansible Deployment Steps
-1. Change working directory to ansible
+
+1. At this stage we'll be using Ansible to configure the VMs that we
```

Comment:
The first statement here could probably go before all the steps are outlined as a general intro to the section.